bot/src/test/java/de/chojo/gamejam/data/wrapper/jam/TimeFrameTest.java

```diff
@@ -0,0 +1,57 @@
+/*
+ *     SPDX-License-Identifier: AGPL-3.0-only
+ *
+ *     Copyright (C) 2022 DevCord Team and Contributor
+ */
+
+package de.chojo.gamejam.data.wrapper.jam;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+import java.time.LocalDateTime;
+import java.time.ZoneId;
+import java.time.ZoneOffset;
+import java.time.ZonedDateTime;
+import java.time.format.DateTimeFormatter;
+import java.time.temporal.TemporalAccessor;
+
+class TimeFrameTest {
+    private static final ZoneId berlin = ZoneId.of("Europe/Berlin");
+    private static final ZoneId newyork = ZoneId.of("America/New_York");
+
+    @Test
+    void equals() {
+        Assertions.assertEquals(TimeFrame.fromEpoch(0, 1000, berlin), TimeFrame.fromEpoch(0, 1000, berlin));
+        Assertions.assertNotEquals(TimeFrame.fromEpoch(0, 1500, berlin), TimeFrame.fromEpoch(0, 1000, berlin));
+    }
+
+    @Test
+    void contains() {
```

Comment:
That doesn't seems to be a valid test :D

Reply:
All those tests are currently for code testing understanding the zoned time clusterfuck. Didnt wanted to push this class yet.

Reply:
Ah, then it fits